src/compiler/scala/tools/nsc/typechecker/Typers.scala

```diff
@@ -5291,12 +5295,12 @@ trait Typers extends Adaptations with Tags with TypersTracking with PatternTyper
           // however we cannot do `context.withMacrosDisabled`
           // because `expr` might contain nested macro calls (see scala/bug#6673)
           //
-          // Note: apparently `Function(Nil, EmptyTree)` is the secret parser marker
+          // Note: `Function(Nil, EmptyTree)` is the secret parser marker
           // which means trailing underscore -- denoting a method value. See makeMethodValue in TreeBuilder.
```

Comment:
This comment could be removed now IMO, perhaps replaced with a one line doc comment on `MethodValue` to show the syntactic form it represents.

Reply:
Yes.